.cico/cico_openshift_e2e.sh

```diff
@@ -30,18 +30,9 @@ if ! hash operator-sdk 2>/dev/null; then
         rm operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu
 fi
 
-# Add kubernetes-api CRDS
-make update_devworkspace_crds
-
-# Install go modules
+# For some reason go on PROW force usage vendor folder
+# This workaround is here until we don't figure out cause
```

Comment:
I supposed that `export GO111MODULE=on` could help but it did not helped:
<details>

<summary>PROW test run logs</summary>

```
secret/support created
+ export CI=openshift
+ CI=openshift
+ export ARTIFACTS_DIR=/tmp/artifacts
+ ARTIFACTS_DIR=/tmp/artifacts
+ export 'GIT_COMMITTER_NAME=CI BOT'
+ GIT_COMMITTER_NAME='CI BOT'
+ export GIT_COMMITTER_EMAIL=ci_bot@notused.com
+ GIT_COMMITTER_EMAIL=ci_bot@notused.com
+ hash operator-sdk
+ mkdir -p /go/bin
++ pwd
+ export PATH=/tmp/shared:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin:/go/bin:/go/src/github.com/devfile/devworkspace-operator:/go/bin
+ PATH=/tmp/shared:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin:/go/bin:/go/src/github.com/devfile/devworkspace-operator:/go/bin
+ OPERATOR_SDK_VERSION=v0.17.0
+ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/v0.17.0/operator-sdk-v0.17.0-x86_64-linux-gnu
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   650  100   650    0     0   2714      0 --:--:-- --:--:-- --:--:--  2708
100 66.6M  100 66.6M    0     0  65.8M      0  0:00:01  0:00:01 --:--:-- 65.8M
+ chmod +x operator-sdk-v0.17.0-x86_64-linux-gnu
+ cp operator-sdk-v0.17.0-x86_64-linux-gnu /go/bin/operator-sdk
+ rm operator-sdk-v0.17.0-x86_64-linux-gnu
+ export GO111MODULE=on
+ GO111MODULE=on
+ make test_e2e
Current env vars:
    NAMESPACE=devworkspace-controller
    IMG=quay.io/che-incubator/che-workspace-controller:nightly
    PULL_POLICY=Always
    ROUTING_SUFFIX=192.168.99.100.nip.io
    WEBHOOK_ENABLED=true
    DEFAULT_ROUTING=basic
    REGISTRY_ENABLED=true
    DEVWORKSPACE_API_VERSION=master
    CERT_IMG=quay.io/che-incubator/che-workspace-controller-cert-gen:nightly
Error from server (NotFound): namespaces "devworkspace-controller" not found
sed -i.bak -e "s|controller.plugin_registry.url: .*|controller.plugin_registry.url: http://|g" ./deploy/controller_config.yaml
sed -i.bak -e 's|controller.webhooks.enabled: .*|controller.webhooks.enabled: "true"|g' ./deploy/controller_config.yaml
sed -i.bak -e 's|devworkspace.default_routing_class: .*|devworkspace.default_routing_class: "basic"|g' ./deploy/controller_config.yaml
sed -i.bak -e 's|devworkspace.routing.cluster_host_suffix: .*|devworkspace.routing.cluster_host_suffix: 192.168.99.100.nip.io|g' ./deploy/controller_config.yaml
sed -i.bak -e 's|devworkspace.sidecar.image_pull_policy: .*|devworkspace.sidecar.image_pull_policy: Always|g' ./deploy/controller_config.yaml
rm ./deploy/controller_config.yaml.bak
sed -i.bak -e "s|image: .*|image: quay.io/che-incubator/che-workspace-controller:nightly|g" ./deploy/os/controller.yaml
sed -i.bak -e "s|imagePullPolicy: Always|imagePullPolicy: Always|g" ./deploy/os/controller.yaml
sed -i.bak -e "s|kubectl.kubernetes.io/restartedAt: .*|kubectl.kubernetes.io/restartedAt: '$(date +%Y-%m-%dT%H:%M:%S%z)'|g" ./deploy/os/controller.yaml
sed -i.bak -e "s|image: .*|image: quay.io/che-incubator/che-workspace-controller-cert-gen:nightly|g" ./deploy/os/devworkspace-controller-cert-gen-deployment.yaml
sed -i.bak -e "s|kubectl.kubernetes.io/restartedAt: .*|kubectl.kubernetes.io/restartedAt: '$(date +%Y-%m-%dT%H:%M:%S%z)'|g" ./deploy/os/devworkspace-controller-cert-gen-deployment.yaml
rm ./deploy/os/controller.yaml.bak
rm ./deploy/os/devworkspace-controller-cert-gen-deployment.yaml.bak
Initialized empty Git repository in /go/src/github.com/devfile/devworkspace-operator/devworkspace-crds/.git/
Updating origin
From https://github.com/devfile/kubernetes-api
 * [new branch]      Implement-agreed-on-devfile-2.0-issues -> origin/Implement-agreed-on-devfile-2.0-issues
 * [new branch]      add-events -> origin/add-events
 * [new branch]      add-test-link-in-readme -> origin/add-test-link-in-readme
 * [new branch]      back-to-workspace-apigroup-prefix -> origin/back-to-workspace-apigroup-prefix
 * [new branch]      fix-devfile -> origin/fix-devfile
 * [new branch]      fix-devfile-project -> origin/fix-devfile-project
 * [new branch]      l0rd-patch-1 -> origin/l0rd-patch-1
 * [new branch]      master     -> origin/master
 * [new branch]      proposal-25-variant-1-define-stacks -> origin/proposal-25-variant-1-define-stacks
 * [new branch]      proposal-31-variant1-better-plugin-mechanism -> origin/proposal-31-variant1-better-plugin-mechanism
 * [new branch]      run-as-user-issue -> origin/run-as-user-issue
 * [new branch]      sleshche-test -> origin/sleshche-test
 * [new branch]      test-documentation -> origin/test-documentation
 * [new branch]      use-devfile-2.0-in-workspace-controller -> origin/use-devfile-2.0-in-workspace-controller
DevWorkspace API is specified from branch
Already on 'master'
Branch master set up to track remote branch master from origin.
HEAD is now at d65a844 Fix wrong naming in Command Group Go definitions (#94)
go mod download
CGO_ENABLED=0 go test -v -c -o bin/devworkspace-controller-e2e ./test/e2e/cmd/workspaces_test.go
./bin/devworkspace-controller-e2e
go: finding bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898
go: finding bou.ke/monkey v1.0.1
go: finding cloud.google.com/go v0.49.0
go: finding cloud.google.com/go/bigquery v1.3.0
go: finding cloud.google.com/go/datastore v1.0.0
go: finding cloud.google.com/go/pubsub v1.0.1
go: finding cloud.google.com/go/storage v1.3.0
go: finding contrib.go.opencensus.io/exporter/ocagent v0.6.0
go: finding dmitri.shuralyov.com/gpu/mtl v0.0.0-20190408044501-666a987793e9
go: finding github.com/Azure/azure-pipeline-go v0.2.2
go: finding github.com/Azure/azure-sdk-for-go v36.1.0+incompatible
go: finding github.com/Azure/azure-storage-blob-go v0.8.0
go: finding github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78
go: finding github.com/Azure/go-autorest v13.3.2+incompatible
go: finding github.com/Azure/go-autorest/autorest v0.9.3-0.20191028180845-3492b2aff503
go: finding github.com/Azure/go-autorest/autorest/adal v0.8.1-0.20191028180845-3492b2aff503
go: finding github.com/Azure/go-autorest/autorest/date v0.2.0
go: finding github.com/Azure/go-autorest/autorest/mocks v0.3.0
go: finding github.com/Azure/go-autorest/autorest/to v0.3.1-0.20191028180845-3492b2aff503
go: finding github.com/Azure/go-autorest/autorest/validation v0.2.1-0.20191028180845-3492b2aff503
go: finding github.com/Azure/go-autorest/logger v0.1.0
go: finding github.com/Azure/go-autorest/tracing v0.5.0
go: finding github.com/BurntSushi/toml v0.3.1
go: finding github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802
go: finding github.com/DATA-DOG/go-sqlmock v1.4.1
go: finding github.com/DataDog/datadog-go v3.2.0+incompatible
go: finding github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd
go: finding github.com/Masterminds/goutils v1.1.0
go: finding github.com/Masterminds/semver v1.5.0
go: finding github.com/Masterminds/semver/v3 v3.0.3
go: finding github.com/Masterminds/sprig/v3 v3.0.2
go: finding github.com/Masterminds/vcs v1.13.1
go: finding github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5
go: finding github.com/Microsoft/hcsshim v0.8.7
go: finding github.com/NYTimes/gziphandler v1.1.1
go: finding github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5
go: finding github.com/OneOfOne/xxhash v1.2.6
go: finding github.com/PuerkitoBio/purell v1.1.1
go: finding github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578
go: finding github.com/Shopify/logrus-bugsnag v0.0.0-20171204204709-577dee27f20d
go: finding github.com/Shopify/sarama v1.19.0
go: finding github.com/Shopify/toxiproxy v2.1.4+incompatible
go: finding github.com/agnivade/levenshtein v1.0.1
go: finding github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
go: finding github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d
go: finding github.com/aliyun/aliyun-oss-go-sdk v2.0.4+incompatible
go: finding github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883
go: finding github.com/antihax/optional v0.0.0-20180407024304-ca021399b1a6
go: finding github.com/apache/thrift v0.12.0
go: finding github.com/armon/circbuf v0.0.0-20150827004946-bbbad097214e
go: finding github.com/armon/consul-api v0.0.0-20180202201655-eb2c6b5be1b6
go: finding github.com/armon/go-metrics v0.3.0
go: finding github.com/armon/go-radix v1.0.0
go: finding github.com/asaskevich/govalidator v0.0.0-20200108200545-475eaeb16496
go: finding github.com/aws/aws-sdk-go v1.25.48
go: finding github.com/baiyubin/aliyun-sts-go-sdk v0.0.0-20180326062324-cfa1a18b161f
go: finding github.com/beorn7/perks v1.0.1
go: finding github.com/bgentry/speakeasy v0.1.0
go: finding github.com/bitly/go-hostpool v0.0.0-20171023180738-a3a6125de932
go: finding github.com/bitly/go-simplejson v0.5.0
go: finding github.com/blang/semver v3.5.1+incompatible
go: finding github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869
go: finding github.com/bradfitz/gomemcache v0.0.0-20190913173617-a41fca850d0b
go: finding github.com/brancz/gojsontoyaml v0.0.0-20190425155809-e8bd32d46b3d
go: finding github.com/bshuster-repo/logrus-logstash-hook v0.4.1
go: finding github.com/bugsnag/bugsnag-go v0.0.0-20141110184014-b1d153021fcd
go: finding github.com/bugsnag/osext v0.0.0-20130617224835-0dd3f918b21b
go: finding github.com/bugsnag/panicwrap v0.0.0-20151223152923-e2c28503fcd0
go: finding github.com/campoy/embedmd v1.0.0
go: finding github.com/cenkalti/backoff v0.0.0-20181003080854-62661b46c409
go: finding github.com/census-instrumentation/opencensus-proto v0.2.1
go: finding github.com/cespare/xxhash v1.1.0
go: finding github.com/cespare/xxhash/v2 v2.1.1
go: finding github.com/chai2010/gettext-go v0.0.0-20160711120539-c6fed771bfd5
go: finding github.com/circonus-labs/circonus-gometrics v2.3.1+incompatible
go: finding github.com/circonus-labs/circonusllhist v0.1.3
go: finding github.com/client9/misspell v0.3.4
go: finding github.com/cockroachdb/apd v1.1.0
go: finding github.com/cockroachdb/cockroach-go v0.0.0-20181001143604-e0a95dfd547c
go: finding github.com/cockroachdb/datadriven v0.0.0-20190809214429-80d97fb3cbaa
go: finding github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd
go: finding github.com/containerd/cgroups v0.0.0-20190919134610-bf292b21730f
go: finding github.com/containerd/console v0.0.0-20180822173158-c12b1e7919c1
go: finding github.com/containerd/containerd v1.3.2
go: finding github.com/containerd/continuity v0.0.0-20200107194136-26c1120b8d41
go: finding github.com/containerd/fifo v0.0.0-20190226154929-a9fb20d87448
go: finding github.com/containerd/go-runc v0.0.0-20180907222934-5a6d9f37cfa3
go: finding github.com/containerd/ttrpc v0.0.0-20190828154514-0e0f228740de
go: finding github.com/containerd/typeurl v0.0.0-20180627222232-a93fcdb778cd
go: finding github.com/coreos/bbolt v1.3.3
go: finding github.com/coreos/etcd v3.3.17+incompatible
go: finding github.com/coreos/go-etcd v2.0.0+incompatible
go: finding github.com/coreos/go-oidc v2.1.0+incompatible
go: finding github.com/coreos/go-semver v0.3.0
go: finding github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
go: finding github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f
go: finding github.com/coreos/prometheus-operator v0.38.0
go: finding github.com/cpuguy83/go-md2man v1.0.10
go: finding github.com/creack/pty v1.1.7
go: finding github.com/cyphar/filepath-securejoin v0.2.2
go: finding github.com/cznic/b v0.0.0-20180115125044-35e9bbe41f07
go: finding github.com/cznic/fileutil v0.0.0-20180108211300-6a051e75936f
go: finding github.com/cznic/golex v0.0.0-20170803123110-4ab7c5e190e4
go: finding github.com/cznic/internal v0.0.0-20180608152220-f44710a21d00
go: finding github.com/cznic/lldb v1.1.0
go: finding github.com/cznic/mathutil v0.0.0-20180504122225-ca4c9f2c1369
go: finding github.com/cznic/ql v1.2.0
go: finding github.com/cznic/sortutil v0.0.0-20150617083342-4c7342852e65
go: finding github.com/cznic/strutil v0.0.0-20171016134553-529a34b1c186
go: finding github.com/cznic/zappy v0.0.0-20160723133515-2533cb5b45cc
go: finding github.com/davecgh/go-spew v1.1.1
go: finding github.com/daviddengcn/go-colortext v0.0.0-20160507010035-511bcaf42ccd
go: finding github.com/deislabs/oras v0.8.1
go: finding github.com/denisenkom/go-mssqldb v0.0.0-20190515213511-eb9f6a1743f3
go: finding github.com/denverdino/aliyungo v0.0.0-20190125010748-a747050bb1ba
go: finding github.com/devfile/kubernetes-api v0.0.0-20200610142627-e6a761a9ecb3
go: finding github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: finding github.com/dgryski/go-sip13 v0.0.0-20190329191031-25c5027a8c7b
go: finding github.com/dhui/dktest v0.3.0
go: finding github.com/dnaeon/go-vcr v1.0.1
go: finding github.com/docker/cli v0.0.0-20200130152716-5d0cf8839492
go: finding github.com/docker/distribution v2.7.1+incompatible
go: finding github.com/docker/docker v1.4.2-0.20200203170920-46ec8731fbce
go: finding github.com/docker/docker-credential-helpers v0.6.3
go: finding github.com/docker/go-connections v0.4.0
go: finding github.com/docker/go-metrics v0.0.0-20180209012529-399ea8c73916
go: finding github.com/docker/go-units v0.4.0
go: finding github.com/docker/libtrust v0.0.0-20150114040149-fa567046d9b1
go: finding github.com/docker/spdystream v0.0.0-20160310174837-449fdfce4d96
go: finding github.com/docopt/docopt-go v0.0.0-20180111231733-ee0de3bc6815
go: finding github.com/dustin/go-humanize v1.0.0
go: finding github.com/eapache/go-resiliency v1.1.0
go: finding github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: finding github.com/eapache/queue v1.1.0
go: finding github.com/eclipse/che-go-jsonrpc v0.0.0-20181205102516-87cdb8da2597
go: finding github.com/eclipse/che-plugin-broker v3.1.1-0.20200207223144-b20597f15e4c+incompatible
go: finding github.com/edsrzf/mmap-go v1.0.0
go: finding github.com/elastic/go-sysinfo v1.1.1
go: finding github.com/elastic/go-windows v1.0.1
go: finding github.com/elazarl/goproxy v0.0.0-20170405201442-c4fc26588b6e
go: finding github.com/emicklei/go-restful v2.9.5+incompatible
go: finding github.com/envoyproxy/go-control-plane v0.9.1-0.20191026205805-5f8ba28d4473
go: finding github.com/envoyproxy/protoc-gen-validate v0.1.0
go: finding github.com/evanphx/json-patch v4.5.0+incompatible
go: finding github.com/exponent-io/jsonpath v0.0.0-20151013193312-d6023ce2651d
go: finding github.com/facette/natsort v0.0.0-20181210072756-2cd4dd1e2dcb
go: finding github.com/fatih/camelcase v1.0.0
go: finding github.com/fatih/color v1.7.0
go: finding github.com/fatih/structtag v1.1.0
go: finding github.com/fortytw2/leaktest v1.3.0
go: finding github.com/fsnotify/fsnotify v1.4.7
go: finding github.com/fsouza/fake-gcs-server v1.7.0
go: finding github.com/garyburd/redigo v0.0.0-20150301180006-535138d7bcd7
go: finding github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
go: finding github.com/globalsign/mgo v0.0.0-20181015135952-eeefdecb41b8
go: finding github.com/go-bindata/go-bindata v3.1.2+incompatible
go: finding github.com/go-gl/glfw v0.0.0-20190409004039-e6da0acd62b1
go: finding github.com/go-ini/ini v1.25.4
go: finding github.com/go-kit/kit v0.9.0
go: finding github.com/go-logfmt/logfmt v0.4.0
go: finding github.com/go-logr/logr v0.1.0
go: finding github.com/go-logr/zapr v0.1.1
go: finding github.com/go-openapi/analysis v0.19.5
go: finding github.com/go-openapi/errors v0.19.2
go: finding github.com/go-openapi/jsonpointer v0.19.3
go: finding github.com/go-openapi/jsonreference v0.19.3
go: finding github.com/go-openapi/loads v0.19.4
go: finding github.com/go-openapi/runtime v0.19.4
go: finding github.com/go-openapi/spec v0.19.4
go: finding github.com/go-openapi/strfmt v0.19.3
go: finding github.com/go-openapi/swag v0.19.5
go: finding github.com/go-openapi/validate v0.19.5
go: finding github.com/go-sql-driver/mysql v1.4.1
go: finding github.com/go-stack/stack v1.8.0
go: finding github.com/gobuffalo/envy v1.7.0
go: finding github.com/gobuffalo/flect v0.2.0
go: finding github.com/gobuffalo/logger v1.0.0
go: finding github.com/gobuffalo/packd v0.3.0
go: finding github.com/gobuffalo/packr v1.30.1
go: finding github.com/gobuffalo/packr/v2 v2.5.1
go: finding github.com/gobwas/glob v0.2.3
go: finding github.com/gocql/gocql v0.0.0-20190301043612-f6df8288f9b4
go: finding github.com/godbus/dbus v0.0.0-20190422162347-ade71ed3457e
go: finding github.com/gofrs/flock v0.7.1
go: finding github.com/gogo/protobuf v1.3.1
go: finding github.com/golang-migrate/migrate/v4 v4.6.2
go: finding github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
go: finding github.com/golang/groupcache v0.0.0-20191027212112-611e8accdfc9
go: finding github.com/golang/mock v1.3.1
go: finding github.com/golang/protobuf v1.3.2
go: finding github.com/golang/snappy v0.0.1
go: finding github.com/golangplus/bytes v0.0.0-20160111154220-45c989fe5450
go: finding github.com/golangplus/fmt v0.0.0-20150411045040-2a5d6d7d2995
go: finding github.com/golangplus/testing v0.0.0-20180327235837-af21d9c3145e
go: finding github.com/google/btree v1.0.0
go: finding github.com/google/go-cmp v0.4.0
go: finding github.com/google/go-github v17.0.0+incompatible
go: finding github.com/google/go-querystring v1.0.0
go: finding github.com/google/gofuzz v1.0.0
go: finding github.com/google/martian v2.1.0+incompatible
go: finding github.com/google/pprof v0.0.0-20190723021845-34ac40c74b70
go: finding github.com/google/renameio v0.1.0
go: finding github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
go: finding github.com/google/uuid v1.1.1
go: finding github.com/googleapis/gax-go v2.0.2+incompatible
go: finding github.com/googleapis/gax-go/v2 v2.0.5
go: finding github.com/googleapis/gnostic v0.3.1
go: finding github.com/gophercloud/gophercloud v0.6.0
go: finding github.com/gopherjs/gopherjs v0.0.0-20191106031601-ce3c9ade29de
go: finding github.com/gorilla/context v1.1.1
go: finding github.com/gorilla/handlers v0.0.0-20150720190736-60c7bfde3e33
go: finding github.com/gorilla/mux v1.7.2
go: finding github.com/gorilla/websocket v1.4.0
go: finding github.com/gosuri/uitable v0.0.4
go: finding github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7
go: finding github.com/grpc-ecosystem/go-grpc-middleware v1.1.0
go: finding github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
go: finding github.com/grpc-ecosystem/grpc-gateway v1.12.1
go: finding github.com/grpc-ecosystem/grpc-health-probe v0.2.1-0.20181220223928-2bf0a5b182db
go: finding github.com/hailocab/go-hostpool v0.0.0-20160125115350-e80d13ce29ed
go: finding github.com/hashicorp/consul/api v1.3.0
go: finding github.com/hashicorp/consul/sdk v0.3.0
go: finding github.com/hashicorp/errwrap v1.0.0
go: finding github.com/hashicorp/go-cleanhttp v0.5.1
go: finding github.com/hashicorp/go-immutable-radix v1.1.0
go: finding github.com/hashicorp/go-msgpack v0.5.5
go: finding github.com/hashicorp/go-multierror v1.0.0
go: finding github.com/hashicorp/go-retryablehttp v0.5.3
go: finding github.com/hashicorp/go-rootcerts v1.0.1
go: finding github.com/hashicorp/go-sockaddr v1.0.2
go: finding github.com/hashicorp/go-syslog v1.0.0
go: finding github.com/hashicorp/go-uuid v1.0.1
go: finding github.com/hashicorp/go-version v1.1.0
go: finding github.com/hashicorp/go.net v0.0.1
go: finding github.com/hashicorp/golang-lru v0.5.3
go: finding github.com/hashicorp/hcl v1.0.0
go: finding github.com/hashicorp/logutils v1.0.0
go: finding github.com/hashicorp/mdns v1.0.0
go: finding github.com/hashicorp/memberlist v0.1.5
go: finding github.com/hashicorp/serf v0.8.5
go: finding github.com/helm/helm-2to3 v0.5.1
go: finding github.com/hpcloud/tail v1.0.0
go: finding github.com/huandu/xstrings v1.2.0
go: finding github.com/iancoleman/strcase v0.0.0-20190422225806-e506e3ef7365
go: finding github.com/imdario/mergo v0.3.7
go: finding github.com/inconshreveable/mousetrap v1.0.0
go: finding github.com/influxdata/influxdb v1.7.7
go: finding github.com/jackc/fake v0.0.0-20150926172116-812a484cc733
go: finding github.com/jackc/pgx v3.2.0+incompatible
go: finding github.com/jessevdk/go-flags v1.4.0
go: finding github.com/jmespath/go-jmespath v0.0.0-20180206201540-c2b33e8439af
go: finding github.com/jmoiron/sqlx v1.2.0
go: finding github.com/joefitzgerald/rainbow-reporter v0.1.0
go: finding github.com/joeshaw/multierror v0.0.0-20140124173710-69b34d4ec901
go: finding github.com/joho/godotenv v1.3.0
go: finding github.com/jonboulle/clockwork v0.1.0
go: finding github.com/jpillora/backoff v1.0.0
go: finding github.com/json-iterator/go v1.1.9
go: finding github.com/jsonnet-bundler/jsonnet-bundler v0.2.0
go: finding github.com/jstemmer/go-junit-report v0.9.1
go: finding github.com/jtolds/gls v4.20.0+incompatible
go: finding github.com/julienschmidt/httprouter v1.3.0
go: finding github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0
go: finding github.com/karrick/godirwalk v1.10.12
go: finding github.com/kisielk/errcheck v1.2.0
go: finding github.com/kisielk/gotool v1.0.0
go: finding github.com/konsorten/go-windows-terminal-sequences v1.0.2
go: finding github.com/kr/logfmt v0.0.0-20140226030751-b84e30acd515
go: finding github.com/kr/pretty v0.1.0
go: finding github.com/kr/pty v1.1.5
go: finding github.com/kr/text v0.1.0
go: finding github.com/kshvakov/clickhouse v1.3.5
go: finding github.com/kylelemons/godebug v0.0.0-20170820004349-d65d576e9348
go: finding github.com/leanovate/gopter v0.2.4
go: finding github.com/lib/pq v1.0.0
go: finding github.com/liggitt/tabwriter v0.0.0-20181228230101-89fcab3d43de
go: finding github.com/lightstep/lightstep-tracer-common/golang/gogo v0.0.0-20190605223551-bc2310a04743
go: finding github.com/lightstep/lightstep-tracer-go v0.18.0
go: finding github.com/lithammer/dedent v1.1.0
go: finding github.com/lovoo/gcloud-opentracing v0.3.0
go: finding github.com/magiconair/properties v1.8.0
go: finding github.com/mailru/easyjson v0.7.0
go: finding github.com/maorfr/helm-plugin-utils v0.0.0-20200216074820-36d2fcf6ae86
go: finding github.com/markbates/inflect v1.0.4
go: finding github.com/marstr/guid v1.1.0
go: finding github.com/martinlindhe/base36 v1.0.0
go: finding github.com/mattn/go-colorable v0.1.2
go: finding github.com/mattn/go-ieproxy v0.0.0-20191113090002-7c0f6868bffe
go: finding github.com/mattn/go-isatty v0.0.12
go: finding github.com/mattn/go-runewidth v0.0.6
go: finding github.com/mattn/go-shellwords v1.0.9
go: finding github.com/mattn/go-sqlite3 v1.10.0
go: finding github.com/matttproud/golang_protobuf_extensions v1.0.1
go: finding github.com/maxbrunsfeld/counterfeiter/v6 v6.2.2
go: finding github.com/miekg/dns v1.1.22
go: finding github.com/mikefarah/yaml/v2 v2.4.0
go: finding github.com/mikefarah/yq/v2 v2.4.1
go: finding github.com/minio/minio-go/v6 v6.0.49
go: finding github.com/minio/sha256-simd v0.1.1
go: finding github.com/mitchellh/cli v1.0.0
go: finding github.com/mitchellh/copystructure v1.0.0
go: finding github.com/mitchellh/go-homedir v1.1.0
go: finding github.com/mitchellh/go-testing-interface v1.0.0
go: finding github.com/mitchellh/go-wordwrap v1.0.0
go: finding github.com/mitchellh/gox v0.4.0
go: finding github.com/mitchellh/hashstructure v1.0.0
go: finding github.com/mitchellh/iochan v1.0.0
go: finding github.com/mitchellh/mapstructure v1.1.2
go: finding github.com/mitchellh/osext v0.0.0-20151018003038-5e2d6d41470f
go: finding github.com/mitchellh/reflectwalk v1.0.0
go: finding github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: finding github.com/modern-go/reflect2 v1.0.1
go: finding github.com/morikuni/aec v1.0.0
go: finding github.com/mozillazg/go-cos v0.13.0
go: finding github.com/mozillazg/go-httpheader v0.2.1
go: finding github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
go: finding github.com/mwitkow/go-conntrack v0.0.0-20190716064945-2f068394615f
go: finding github.com/mxk/go-flowrate v0.0.0-20140419014527-cca7078d478f
go: finding github.com/nakagami/firebirdsql v0.0.0-20190310045651-3c02a58cfed8
go: finding github.com/ncw/swift v1.0.47
go: finding github.com/oklog/run v1.0.0
go: finding github.com/oklog/ulid v1.3.1
go: finding github.com/olekukonko/tablewriter v0.0.2
go: finding github.com/onsi/ginkgo v1.11.0
go: finding github.com/onsi/gomega v1.8.1
go: finding github.com/opencontainers/go-digest v1.0.0-rc1
go: finding github.com/opencontainers/image-spec v1.0.1
go: finding github.com/opencontainers/runc v0.1.1
go: finding github.com/opencontainers/runtime-spec v0.1.2-0.20190507144316-5b71a03e2700
go: finding github.com/opencontainers/runtime-tools v0.0.0-20181011054405-1d69bd0f9c39
go: finding github.com/openshift/api v3.9.1-0.20190424152011-77b8897ec79a+incompatible
go: finding github.com/openshift/client-go v0.0.0-20190923180330-3b6373338c9b
go: finding github.com/openshift/origin v0.0.0-20160503220234-8f127d736703
go: finding github.com/openshift/prom-label-proxy v0.1.1-0.20191016113035-b8153a7f39f1
go: finding github.com/opentracing-contrib/go-stdlib v0.0.0-20190519235532-cf7a6c988dc9
go: finding github.com/opentracing/basictracer-go v1.0.0
go: finding github.com/opentracing/opentracing-go v1.1.0
go: finding github.com/openzipkin/zipkin-go v0.1.6
go: finding github.com/operator-framework/api v0.1.1
go: finding github.com/operator-framework/operator-lifecycle-manager v0.0.0-20200321030439-57b580e57e88
go: finding github.com/operator-framework/operator-registry v1.6.2-0.20200330184612-11867930adb5
go: finding github.com/operator-framework/operator-sdk v0.17.0
go: finding github.com/otiai10/copy v1.0.2
go: finding github.com/otiai10/curr v0.0.0-20190513014714-f5a3d24e5776
go: finding github.com/otiai10/mint v1.3.0
go: finding github.com/pascaldekloe/goe v0.1.0
go: finding github.com/pborman/uuid v1.2.0
go: finding github.com/pelletier/go-toml v1.2.0
go: finding github.com/peterbourgon/diskv v2.0.1+incompatible
go: finding github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
go: finding github.com/pierrec/lz4 v2.0.5+incompatible
go: finding github.com/pkg/errors v0.9.1
go: finding github.com/pmezard/go-difflib v1.0.0
go: finding github.com/posener/complete v1.1.1
go: finding github.com/pquerna/cachecontrol v0.0.0-20171018203845-0dec1b30a021
go: finding github.com/prometheus/alertmanager v0.20.0
go: finding github.com/prometheus/client_golang v1.5.1
go: finding github.com/prometheus/client_model v0.2.0
go: finding github.com/prometheus/common v0.9.1
go: finding github.com/prometheus/procfs v0.0.8
go: finding github.com/prometheus/prometheus v2.3.2+incompatible
go: finding github.com/prometheus/tsdb v0.7.1
go: finding github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a
go: finding github.com/remyoudompheng/bigfft v0.0.0-20170806203942-52369c62f446
go: finding github.com/robfig/cron v0.0.0-20170526150127-736158dc09e1
go: finding github.com/rogpeppe/fastuuid v1.2.0
go: finding github.com/rogpeppe/go-internal v1.5.0
go: finding github.com/rs/cors v1.6.0
go: finding github.com/rubenv/sql-migrate v0.0.0-20191025130928-9355dd04f4b3
go: finding github.com/russross/blackfriday v1.5.2
go: finding github.com/ryanuber/columnize v2.1.0+incompatible
go: finding github.com/samuel/go-zookeeper v0.0.0-20190923202752-2cc03de413da
go: finding github.com/santhosh-tekuri/jsonschema v1.2.4
go: finding github.com/satori/go.uuid v1.2.0
go: finding github.com/sclevine/spec v1.2.0
go: finding github.com/sean-/seed v0.0.0-20170313163322-e2103e2c3529
go: finding github.com/sergi/go-diff v1.0.0
go: finding github.com/shopspring/decimal v0.0.0-20180709203117-cd690d0c9e24
go: finding github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749
go: finding github.com/shurcooL/vfsgen v0.0.0-20181202132449-6a9ea43bcacd
go: finding github.com/sirupsen/logrus v1.5.0
go: finding github.com/smartystreets/assertions v1.0.1
go: finding github.com/smartystreets/goconvey v1.6.4
go: finding github.com/soheilhy/cmux v0.1.4
go: finding github.com/spaolacci/murmur3 v1.1.0
go: finding github.com/spf13/afero v1.2.2
go: finding github.com/spf13/cast v1.3.0
go: finding github.com/spf13/cobra v0.0.5
go: finding github.com/spf13/jwalterweatherman v1.0.0
go: finding github.com/spf13/pflag v1.0.5
go: finding github.com/spf13/viper v1.4.0
go: finding github.com/stretchr/objx v0.2.0
go: finding github.com/stretchr/testify v1.4.0
go: finding github.com/syndtr/gocapability v0.0.0-20170704070218-db04d3cc01c8
go: finding github.com/thanos-io/thanos v0.11.0
go: finding github.com/tidwall/pretty v1.0.0
go: finding github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5
go: finding github.com/tv42/httpunix v0.0.0-20150427012821-b75d8614f926
go: finding github.com/uber/jaeger-client-go v2.20.1+incompatible
go: finding github.com/uber/jaeger-lib v2.2.0+incompatible
go: finding github.com/ugorji/go v1.1.4
go: finding github.com/ugorji/go/codec v0.0.0-20181204163529-d75b2dcb6bc8
go: finding github.com/urfave/cli v1.20.0
go: finding github.com/vektah/gqlparser v1.1.2
go: finding github.com/xanzy/go-gitlab v0.15.0
go: finding github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: finding github.com/xdg/stringprep v1.0.0
go: finding github.com/xeipuuv/gojsonpointer v0.0.0-20180127040702-4e3ac2762d5f
go: finding github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415
go: finding github.com/xeipuuv/gojsonschema v1.1.0
go: finding github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2
go: finding github.com/xlab/handysort v0.0.0-20150421192137-fb3537ed64a1
go: finding github.com/xlab/treeprint v0.0.0-20180616005107-d6fb6747feb6
go: finding github.com/xordataexchange/crypt v0.0.3-0.20170626215501-b2862e3d0a77
go: finding github.com/yuin/goldmark v1.1.25
go: finding github.com/yvasiyarov/go-metrics v0.0.0-20140926110328-57bccd1ccd43
go: finding github.com/yvasiyarov/gorelic v0.0.0-20141212073537-a9bba5b9ab50
go: finding github.com/yvasiyarov/newrelic_platform_go v0.0.0-20140908184405-b21fdbd4370f
go: finding github.com/ziutek/mymysql v1.5.4
go: finding gitlab.com/nyarla/go-crypt v0.0.0-20160106005555-d9a5dc2b789b
go: finding go.elastic.co/apm v1.5.0
go: finding go.elastic.co/apm/module/apmhttp v1.5.0
go: finding go.elastic.co/apm/module/apmot v1.5.0
go: finding go.elastic.co/fastjson v1.0.0
go: finding go.etcd.io/bbolt v1.3.3
go: finding go.etcd.io/etcd v0.0.0-20191023171146-3cf2f69b5738
go: finding go.mongodb.org/mongo-driver v1.1.2
go: finding go.opencensus.io v0.22.2
go: finding go.uber.org/atomic v1.6.0
go: finding go.uber.org/automaxprocs v1.2.0
go: finding go.uber.org/multierr v1.5.0
go: finding go.uber.org/tools v0.0.0-20190618225709-2cfd321de3ee
go: finding go.uber.org/zap v1.14.1
go: finding golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
go: finding golang.org/x/exp v0.0.0-20191030013958-a1ab85dbe136
go: finding golang.org/x/image v0.0.0-20190802002840-cff245a6509b
go: finding golang.org/x/lint v0.0.0-20190930215403-16217165b5de
go: finding golang.org/x/mobile v0.0.0-20190719004257-d2bd2a29d028
go: finding golang.org/x/mod v0.2.0
go: finding golang.org/x/net v0.0.0-20200226121028-0de0cce0169b
go: finding golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
go: finding golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
go: finding golang.org/x/sys v0.0.0-20200122134326-e047566fdf82
go: finding golang.org/x/text v0.3.2
go: finding golang.org/x/time v0.0.0-20191024005414-555d28b269f0
go: finding golang.org/x/tools v0.0.0-20200327195553-82bb89366a1e
go: finding golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543
go: finding gomodules.xyz/jsonpatch/v2 v2.0.1
go: finding gomodules.xyz/jsonpatch/v3 v3.0.1
go: finding gomodules.xyz/orderedmap v0.1.0
go: finding gonum.org/v1/gonum v0.0.0-20190331200053-3d26580ed485
go: finding gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e
go: finding google.golang.org/api v0.14.0
go: finding google.golang.org/appengine v1.6.5
go: finding google.golang.org/cloud v0.0.0-20151119220103-975617b05ea8
go: finding google.golang.org/genproto v0.0.0-20191115194625-c23dd37a84c9
go: finding google.golang.org/grpc v1.27.0
go: finding gopkg.in/airbrake/gobrake.v2 v2.0.9
go: finding gopkg.in/alecthomas/kingpin.v2 v2.2.6
go: finding gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15
go: finding gopkg.in/cheggaaa/pb.v1 v1.0.25
go: finding gopkg.in/errgo.v2 v2.1.0
go: finding gopkg.in/fsnotify.v1 v1.4.7
go: finding gopkg.in/fsnotify/fsnotify.v1 v1.4.7
go: finding gopkg.in/gemnasium/logrus-airbrake-hook.v2 v2.1.2
go: finding gopkg.in/gorp.v1 v1.7.2
go: finding gopkg.in/imdario/mergo.v0 v0.3.7
go: finding gopkg.in/inf.v0 v0.9.1
go: finding gopkg.in/ini.v1 v1.51.0
go: finding gopkg.in/natefinch/lumberjack.v2 v2.0.0
go: finding gopkg.in/op/go-logging.v1 v1.0.0-20160211212156-b2cb9fa56473
go: finding gopkg.in/resty.v1 v1.12.0
go: finding gopkg.in/square/go-jose.v2 v2.2.2
go: finding gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
go: finding gopkg.in/yaml.v2 v2.2.8
go: finding gopkg.in/yaml.v3 v3.0.0-20190905181640-827449938966
go: finding gotest.tools v2.2.0+incompatible
go: finding helm.sh/helm/v3 v3.1.2
go: finding honnef.co/go/tools v0.0.1-2019.2.3
go: finding howett.net/plist v0.0.0-20181124034731-591f970eefbb
go: finding k8s.io/api v0.17.4
go: finding k8s.io/apiextensions-apiserver v0.17.4
go: finding k8s.io/apimachinery v0.17.4
go: finding k8s.io/apiserver v0.17.4
go: finding k8s.io/autoscaler v0.0.0-20190607113959-1b4f1855cb8e
go: finding k8s.io/cli-runtime v0.17.4
go: finding k8s.io/client-go v0.17.4
go: finding k8s.io/code-generator v0.17.4
go: finding k8s.io/component-base v0.17.4
go: finding k8s.io/gengo v0.0.0-20191010091904-7fa3014cb28f
go: finding k8s.io/helm v2.16.3+incompatible
go: finding k8s.io/klog v1.0.0
go: finding k8s.io/kube-aggregator v0.17.3
go: finding k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
go: finding k8s.io/kube-state-metrics v1.7.2
go: finding k8s.io/kubectl v0.17.4
go: finding k8s.io/kubernetes v1.13.0
go: finding k8s.io/metrics v0.17.4
go: finding k8s.io/utils v0.0.0-20191114200735-6ca3b61696b6
go: finding modernc.org/cc v1.0.0
go: finding modernc.org/golex v1.0.0
go: finding modernc.org/mathutil v1.0.0
go: finding modernc.org/strutil v1.0.0
go: finding modernc.org/xc v1.0.0
go: finding rsc.io/binaryregexp v0.2.0
go: finding rsc.io/letsencrypt v0.0.3
go: finding sigs.k8s.io/controller-runtime v0.5.2
go: finding sigs.k8s.io/controller-tools v0.2.8
go: finding sigs.k8s.io/kustomize v2.0.3+incompatible
go: finding sigs.k8s.io/structured-merge-diff v1.0.2
go: finding sigs.k8s.io/yaml v1.1.0
go: finding vbom.ml/util v0.0.0-20160121211510-db5cfe13f5cc
# command-line-arguments
package command-line-arguments (test)
	imports github.com/onsi/ginkgo: cannot find package "." in:
	/go/src/github.com/devfile/devworkspace-operator/vendor/github.com/onsi/ginkgo
FAIL	command-line-arguments [setup failed]
FAIL
/bin/sh: line 2: ./bin/devworkspace-controller-e2e: No such file or directory
make: *** [_do_e2e_test] Error 127
```
</details>

Maybe putting modules in go test command could help like:
```
GO111MODULE=on CGO_ENABLED=0 go test -v -c -o bin/devworkspace-controller-e2e ./test/e2e/cmd/workspaces_test.go
```
but I don't have idea why export does not work... BTW it's not the purpose of this PR.

Reply:
few weeks ago I tried something similar but didn't work, prow still try to use vendor folder for some reason